packages/cli/cli/src/commands/upgrade/upgradeGenerator.ts

```diff
@@ -0,0 +1,93 @@
+import { generatorsYml } from "@fern-api/configuration";
+import { GeneratorsConfigurationSchema } from "@fern-api/configuration/src/generators-yml";
+import { Project } from "@fern-api/project-loader";
+import { TaskContext } from "@fern-api/task-context";
+import { APIWorkspace } from "@fern-api/workspace-loader";
+import chalk from "chalk";
+import { writeFile } from "fs/promises";
+import yaml from "js-yaml";
+import { CliContext } from "../../cli-context/CliContext";
+
+async function upgradeSpecificGroupGenerator({
+    context,
+    generator,
+    group,
+    generatorsConfiguration,
+    workspace
+}: {
+    context: TaskContext;
+    generator: string;
+    group: string;
+    generatorsConfiguration: GeneratorsConfigurationSchema;
+    workspace: APIWorkspace;
+}) {
+    const newConfiguration = generatorsYml.upgradeGenerator({
+        generatorName: generator,
+        generatorsConfiguration,
+        groupName: group,
+        context
+    });
+    await writeFile(
+        workspace.generatorsConfiguration?.absolutePathToConfiguration ??
+            generatorsYml.getPathToGeneratorsConfiguration({
+                absolutePathToWorkspace: workspace.absoluteFilepath
+            }),
+        yaml.dump(newConfiguration)
+    );
```

Comment:
not a big deal but worth calling out that i think this will delete comments https://github.com/vellum-ai/vellum-client-generator/blob/a63b5f8ee60c0d8c2e10ce063adeaecc048e7c93/fern/generators.yml#L10-L15

Reply:
ah nice ok will do a different way